docs/people-and-groups/user-provisioning.md

```diff
@@ -0,0 +1,52 @@
+---
+title: User provisioning with SCIM
+---
+
+# User provisioning with SCIM
+
+{% include plans-blockquote.html feature="User provisioning with SCIM" %}
+
+Metabase supports user provisioning via the System for Cross-domain Identity Management (SCIM) protocol. In addition to Single Sign-on (SSO), you can set up user provisioning in Metabase with SCIM to:
+
+- **Decouple authentication from provisioning**. Even though anyone could authenticate with SSO, you may only want some groups to be able to create an account in Metabase.
+- **Support deprovisioning user accounts**. If you deactivate someone from your SSO, SCIM can let Metabase know to deactivate their Metabase account as well.
+
+> For now, Metabase supports SCIM for [Okta](https://developer.okta.com/docs/concepts/scim/) and [Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/architecture/sync-scim).
```

Comment:
On Okta, we support group provisioning.
Groups in Okta are created and populated in Metabase.

On Entra, we support only user provisioning (groups are not created or populated)

Reply:
Yeah, we should mention that you can use it to sync group memberships as well on Okta